Schyntax/Internals/IntermediateRepresentation.cs

```diff
@@ -58,6 +58,14 @@ public class IrGroup
         public bool HasDaysOfMonthExcluded => _daysOfMonthExcluded != null && _daysOfMonthExcluded.Count > 0;
         public List<IrIntegerRange> DaysOfMonthExcluded => _daysOfMonthExcluded ?? (_daysOfMonthExcluded = new List<IrIntegerRange>());
 
+        private List<IrIntegerRange> _daysOfYear;
+        public bool HasDaysOfYear => _daysOfYear != null && _daysOfYear.Count > 0;
+        public List<IrIntegerRange> DaysOfYear => _daysOfYear ?? (this._daysOfYear = new List<IrIntegerRange>());
```

Comment:
You can drop the `this.` here. I'm assuming that was a mistake since you're not using it anywhere else.